include/functions.h

```diff
@@ -1671,7 +1672,7 @@ s16 ActorCutscene_StartAndSetUnkLinkFields(s16 index, Actor* actor);
 s16 ActorCutscene_StartAndSetFlag(s16 index, Actor* actor);
 s16 ActorCutscene_Start(s16 index, Actor* actor);
 s16 ActorCutscene_Stop(s16 index);
-// s16 ActorCutscene_GetCurrentIndex(void);
+s16 ActorCutscene_GetCurrentIndex();
```

Comment:
```suggestion
s16 ActorCutscene_GetCurrentIndex(void);
```
small nit - in *declarations*, `int foo();` means that foo takes an unspecified number of params, and `int foo(void);` means that foo takes no parameters. (in *definitions* there's no difference).